src/components/TopBar/TopBar.scss

```diff
@@ -5,6 +5,13 @@ $icon-size: rem(20px);
 $page-left-alignment-breakpoint-max: 1268px;
 $context-control-extra-spacing: rem(40px);
 $context-control-expand-after: 1400px;
+$top-bar-scroll-shadow: 0 1px 2px var(--p-shadow-from-direct-light);
+$top-bar-scroll-shadow-transition-timing-function: cubic-bezier(
```

Comment:
Too verbose?

Reply:
Because these variables are in the context of the top bar file, you could say something like:

`$shadow-when-scrolling` and I think `$shadow-easing`